packages/notivue/Notifications/notifications.css

```
@@ -47,10 +48,18 @@
    align-items: var(--nv-y-align);
    border-radius: var(--nv-radius, 0);
    border: var(--nv-border-width, 0) solid var(--nv-border, var(--nv-global-border));
+   overflow: hidden;
 
-   box-shadow:
-      var(--nv-shadow, 0 0 rgba(0, 0, 0, 0)),
-      inset var(--tip-width) 0 0 var(--nv-accent);
+   box-shadow: var(--nv-shadow, 0 0 rgba(0, 0, 0, 0));
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)